src/main/java/at/hannibal2/skyhanni/events/item/ItemHoverEvent.kt

```diff
@@ -3,4 +3,4 @@ package at.hannibal2.skyhanni.events.item
 import at.hannibal2.skyhanni.events.LorenzEvent
 import net.minecraft.item.ItemStack
 
-class ItemHoverEvent(val itemStack: ItemStack, val toolTip: List<String>) : LorenzEvent()
+class ItemHoverEvent(val itemStack: ItemStack, val toolTip: MutableList<String>) : LorenzEvent()
```

Comment:
i dont see why we are changing this event when the pr feature doesnt even use this

Reply:
It is used in the currently commented part
However this cannot be used until https://github.com/hannibal002/SkyHanni/pull/1717 is ready to use since it otherwise would conflict with the enchant parsing
If you want this can be changed when I fix Vixids PR but theres little to no reason to do it in this order

Reply:
But changed it rq